Package.swift

```diff
@@ -20,22 +20,6 @@ let package = Package(
   products: [
     .executable(name: "Runner", targets: ["Runner"]),
     .executable(name: "Server", targets: ["Server"]),
-    .library(name: "Database", targets: ["Database"]),
```

Comment:
This is only if other libraries wanna pull PointFree in and import frameworks...for now simpler to not maintain this list.

Reply:
actually is this necessary to get targets in Xcode too? when I dont have these decelerations im not able to build these targets in isolation anymore

Reply:
They all show up to me like this:

<img width="259" alt="image" src="https://user-images.githubusercontent.com/658/74659589-9d8b0500-5162-11ea-8da3-b6e1ca7abcd3.png">

I'll address comments/merge if nothing's blocking, but we can revert this part if it's behaving weirdly.